docs/configuration/intra_node_data_path.md

```diff
@@ -264,3 +264,28 @@ arch.remove_publisher_callback('/pong_node', '/pong', 'timer_callback_1')
 arch.remove_variable_passing('/pong_node', 'subscription_callback_0', 'timer_callback_1')
 arch.update_message_context('/pong_node', '/ping', '/pong', 'UNDEFINED')
 ```
+
+## Handling special cases
+
+### Call `take()` method of Subscription object
+
+In ROS 2, it is possible to suppress calling subscription callback upon receiving a topic,
+and take a message using `take()` method of Subscription object only when the message is needed.
```

Comment:
We suggest that you split the sentences because they seem to be vague about what you want to say. For example, what about the following sentence?
```suggestion
In ROS 2 it is possible to suppress the invocation of subscription callbacks when receiving a topic.
If the callback is not executed, the user can still receive the message using the `take()` method of the Subscription object.
```

Reply:
Fixed in https://github.com/tier4/caret_doc/pull/355/commits/9edecc2ca5513cbbdbc3818cebb1158913b4d6d5.

Reply:
The text was a bit messy as it was intended as a suggestion. Please allow me to correct it. Can I use a sentence like this?
```
In ROS 2, it is possible to suppress calling subscription callback when receiving a topic.
And even if the callback is not executed, the user can still receive the message using the `take()` method of the `Subscription` object.
```

Reply:
OK, fixed in https://github.com/tier4/caret_doc/pull/355/commits/21cf419c3a826a15d1f3891f497a889a4300febd.